CsprSdkStandardTestsNet/Test/Utils/CasperClientProvider.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Casper.Network.SDK;
+using Casper.Network.SDK.SSE;
+
+namespace CsprSdkStandardTestsNet.Test.Utils;
+
+public class CasperClientProvider
+{
+    private static CasperClientProvider _instance;
+    private NetCasperClient _casperService;
+    private ServerEventsClient _eventService;
+    
+    public static  CasperClientProvider GetInstance()
+    {
+        return _instance ??= new CasperClientProvider();
+    }
+    
+    private CasperClientProvider() {
+        try {
+            var properties = new TestProperties();
+            _casperService = new NetCasperClient("http://" + properties.Hostname + ":" + properties.RcpPort + "/rpc");
+        } catch (Exception exception) {
+            throw new Exception(exception.ToString());
+        }
+    }
+
+    public NetCasperClient CasperService
```

Comment:
Consider using auto properties for all getters and setters eg: 
`public NetCasperClient CasperService { get; set; }`